script/bootstrap

```diff
@@ -6,4 +6,9 @@ set -e
 
 cd "$(dirname "$0")/.."
 script/bootstrap_server
-script/bootstrap_frontend
+
+if [ -x "$(command -v npm >/dev/null)" ]; then
+  script/bootstrap_frontend
+else
+  echo "Frontend development not possible without Node/npm"
```

Comment:
This reminds me, we have moved away from `npm` and use Yarn. That should also be a prerequisite I guess… (also `bootstrap_frontend` needs `npm install` replaced with running `yarn`